docs/data/sdks/typescript-browser/index.md

```diff
@@ -12,6 +12,9 @@ The TypeScript Browser SDK lets you send events to Amplitude. This library is op
 !!!info "Browser SDK Resources"
     [:material-github: Github](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-TypeScript/releases) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-TypeScript/)
 
+!!!note "Typescript Broswer SDK versus the Marketing Analytics Browser"
+    If you need both web attribution and page view tracking. Please use Marketing Analytics Browser instead. It extends the Typescript Browser SDK with build in web attribution and page view tracking. Learn more about [Marketing Analytics Browser SDK](../marketing-analytics-browser/).
```

Comment:
I think we should call out the the TS SDK is prefered for technical users. Here's an attempt at copy. cc @caseyamp 
```suggestion
    The TypeScript Browser SDK is our recommended SDK for most users. It's functionality can easy be extended through the use of plugins.
    However, if you want web attribution and page view tracking without any additional setup and please use Marketing Analytics Browser instead. It extends the Typescript Browser SDK with build in web attribution and page view tracking. Learn more about [Marketing Analytics Browser SDK](../marketing-analytics-browser/).
```

Reply:
Thanks for helping me refine the words.